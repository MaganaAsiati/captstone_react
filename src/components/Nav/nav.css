.links {
  list-style-type: none;
  background-color: #ce4176;
  display: flex;
  justify-content: flex-end;
  padding: 15px;
  font-size: 25px;
  width: 100%;
}

.links a {
  text-decoration: none;
  color: rgba(255, 255, 255, 0.866);
  font-weight: bold;
  margin: 10px;
}

.icon {
  display: flex;
  justify-content: space-between;
  gap: 20px;
}

.icon-mic {
  color: white;
}

.title {
  flex: 1;
  color: rgba(255, 255, 255, 0.866);
  font-weight: bold;
}

.evolution {
  color: rgba(219, 214, 214, 0.693);
}

@media screen and (max-width: 990px) {
  .links a {
    width: 100%;
  }
}
