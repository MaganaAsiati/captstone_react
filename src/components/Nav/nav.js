import React from 'react';
import './nav.css';
import { BsMic } from 'react-icons/bs';
import { AiOutlineSetting } from 'react-icons/ai';

function Nav() {
  return (
    <div>
      <ul className="links">
        <li className="title">
          Covid
          <span className="evolution">Evolution &nbsp;&nbsp;</span>
          <span />
        </li>
        <li className="icon">
          <BsMic className="icon-mic" />
          <AiOutlineSetting className="icon-mic" />
        </li>
      </ul>
    </div>
  );
}

export default Nav;
