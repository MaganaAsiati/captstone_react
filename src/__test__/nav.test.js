import React from 'react';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../Redux/configureStore';
import Nav from '../components/Nav/nav';

describe('Nav Component test', () => {
  it('Renders Mission Row', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <Router>
            <Nav />
          </Router>
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
