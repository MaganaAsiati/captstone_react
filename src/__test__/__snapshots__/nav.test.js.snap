// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Component test Renders Mission Row 1`] = `
<div>
  <ul
    className="links"
  >
    <li
      className="title"
    >
      Covid
      <span
        className="evolution"
      >
        Evolution   
      </span>
      <span />
    </li>
    <li
      className="link"
    >
      <a
        aria-current="page"
        className="active"
        href="#/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
  </ul>
</div>
`;
